automation/presubmit-tests/gofmt.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
may I know where do we invoke this file?

Reply:
in the line 68 of the file [.github/workflows/unit_test.yml](https://github.com/sustainable-computing-io/kepler/pull/166/files#diff-012d76a293bfa67c5d65f938b15705520eef288139f3b513cd16266e6f780a4c)